Expensify/App
# Assign CP PRs to the deployer that triggered them

### Details
Adds a new step in the cherry-pick job to assign the deployer to the PR with conflicts that need to be fixed manually.

### Fixed Issues

$ https://github.com/Expensify/App/issues/4036

### Tests/QA Steps

1. Create a PR with changes in a given file.
1. Give the PR the `CP Staging` label.
1. Merge the PR.
1. Create another PR with changes in the same file (to ensure that there will be conflicts on staging).
1. Give the PR the `CP Staging` label.
1. Merge the PR.
1. The CP PR created for the second PR will be created but not automerged, due to conflicts. 
1. The CP PR should be auto-assigned to the author of the original (second) PR.
